### ISSUE TITLE
performance on modelnet40

### ISSUE THREAD

**Issue by YoungSun3Pan** [HuguesTHOMAS/KPConv#61]
@HuguesTHOMAS Hi, thanks for your code. I trained KPconv_rigid on Modelnet40,  and I got 90.7 accuracy (92.9 reported in paper).  I don't know why.
![modelnet40](https://user-images.githubusercontent.com/53615521/74584075-1e35ee00-5009-11ea-9a69-733c79cdb222.PNG)

**Comment by HuguesTHOMAS** [comment]
Hi @YoungSun3Pan,

This score was obtain during validation and with a progressive voting scheme (this is the ''vote validation when you plot convergence). We tried several parameters and optimized to get this score. It is the only score of the paper that is not likely to be reproduced. Our focus is more on semantic segmentation.

Best,
Hugues

**Comment by YoungSun3Pan** [comment]
@HuguesTHOMAS Thanks, i have solved the above problems.

I have a very interesting problem on testing Shapenetpart..

I got stange results  during testing (only 40map) , but normal results during traing (85 map on val set)

**Comment by YoungSun3Pan** [comment]
![image](https://user-images.githubusercontent.com/53615521/74934400-1a4b0700-5421-11ea-9824-7b72f9b6bee7.png)

**Comment by HuguesTHOMAS** [comment]
I think this may just be an orientation issue. The test set is probably aligned in a different direction than the training set. 
Check the input preparation and try to visualize models to check their orientation
